packages/json/CMakeLists.txt

```diff
@@ -25,6 +26,7 @@ add_library(soss-json SHARED
 target_link_libraries(soss-json
   PUBLIC
     soss::core
+    $<$<PLATFORM_ID:Windows>:yaml-cpp>
```

Comment:
I believe `yaml-cpp` is a public dependency of `soss-core`, so it should be inherited automatically by `soss-json`, right?